### PR TITLE
clear up dependencies

### PR DESCRIPTION
**Pull request by atomboulian**
still need version info but it'll help more newbs like me lol.

### CODE REVIEW

**Comment by atomboulian** [comment]
https://github.com/mitevpi/revit-wpf-template/issues/1

**Comment by mitevpi** [comment]
Hi there - thanks for digging into this! I am traveling this week and next, and will have limited connectivity, but I'll review as soon as I can.

**Comment by atomboulian** [comment]
Sounds good! Your template really helped me get started. You’re the best.

File: README.md [hunk 1]
```diff
@@ -12,9 +12,10 @@ WPF Template for Revit Add-Ins including wrapped external methods for execution
 ### Build
 
 1. Clone/download this repository and open the `.sln` at the root of the repository with Microsoft Visual Studio.
-2. Re-link references to `RevitAPI.dll` and others which may be missing.
-3. Build the solution - Building the solution will automatically create and copy the add-in files to the folder for Revit 2019.
-4. Open Revit - Upon opening Revit 2019, there should be a tab called "Template" in Revit, with a button to launch the WPF add-in.
+2. `Install-Package DynamicLanguageRuntime -Version 1.1.0`
```
Comment (mitevpi) [inline README.md:15]: Why is this dependency added? I was able to compile and execute successfully without, but want to make sure it doesn't serve another function.

**Review by mitevpi — COMMENTED** [review]
Why is the [DLR dependency ](https://www.nuget.org/packages/DynamicLanguageRuntime/)added? I was able to compile and execute successfully without, but want to make sure it doesn't serve another function.

**Comment by mitevpi** [comment]
Closing this for now due to inactivity.

**Pull request closed without merging**